src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_command_type.py

```diff
@@ -43,6 +43,8 @@ def keyvault_exception_handler(ex):
                            'The vault may not exist or you may need to flush your DNS cache '
                            'and try again later.')
         raise CLIError(ex)
+    else:
```

Comment:
@tjprescott, this is an optional change, if you feel it has side impact, I can revert/improve. So far this error handler swallows exceptions and hides the real issue.